src/components/ha-area-picker.ts

```diff
@@ -3,7 +3,7 @@ import "@polymer/paper-input/paper-input";
 import "@polymer/paper-item/paper-item";
 import "@polymer/paper-item/paper-item-body";
 import "@polymer/paper-listbox/paper-listbox";
-import "@vaadin/vaadin-combo-box/theme/material/vaadin-combo-box-light";
+import "@vaadin/combo-box/theme/material/vaadin-combo-box-light";
```

Comment:
When no areas, this one looks weird:

![image](https://user-images.githubusercontent.com/1444314/152616934-364ef203-67ac-4bcb-95e2-105418f3d979.png)

Reply:
Old screenshots from https://design.home-assistant.io/#automation/selectors

![image](https://user-images.githubusercontent.com/1444314/152617219-4f030f46-2a8c-4c4e-a4f4-a10e9d40a700.png)

![image](https://user-images.githubusercontent.com/1444314/152617241-ad904886-cb29-44da-a9da-447ec08abc18.png)

![image](https://user-images.githubusercontent.com/1444314/152617254-45939fa1-f6db-450b-be34-365509b1a427.png)